v2/spanner-to-sourcedb/src/test/java/com/google/cloud/teleport/v2/templates/SpannerToSourceDbIT.java

```diff
@@ -206,4 +217,36 @@ private void assertRowInMySQL() throws InterruptedException {
     assertThat(rows.get(0).get("name")).isEqualTo("FF");
     assertThat(rows.get(0).get("from")).isEqualTo("AA");
   }
+
+  @Test
+  public void spannerToMySQLSourceDbMaxColAndTableNameTest()
```

Comment:
Having multiple Tests share the same dataflow pipeline is inherently flaky.
Just in case they don't schedule in parallel, the pipeline would be terminated causing flakiness.